app/src/main/res/layout/screen_patients.xml

```diff
@@ -63,6 +63,10 @@
           android:id="@+id/simpleVideoLayout"
           layout="@layout/simple_video_layout" />
 
+        <include
```

Comment:
I am guessing we will have a different PR for toggling between illustration, video, and update card?

Reply:
yess. That is a different story where everything comes together. [This](https://app.shortcut.com/simpledotorg/story/7236/implement-showing-app-update-nudges-based-on-the-priority-in-patienttabscreen) one.